### ISSUE TITLE
Some wind plants incorrectly unavailable?

### ISSUE THREAD

**Issue by kdayday** [NREL-SIIP/ExtremeSolarTexas#20]
In my manual updates, I'm still setting all wind plants to available because it looks like 6 are written in as unavailable. Should they be automatically available, too?

```
get_name.(get_components(
               RenewableDispatch,
                       system,
               x -> get_prime_mover(x) != PrimeMovers.PVe && get_available(x) == false))
6-element Vector{String}:
 "PANHANDLE_1"
 "PANHANDLE_3"
 "PANHANDLE_2"
 "LENORAH_1"
 "LORAINE_1"
 "FORSAN_2"
```

**Comment by jd-lara** [comment]
I will make this fix

**Comment by jd-lara** [comment]
closed via c96d6ada8d9d6acec9c7e380e95ce4770f5078e2
